pkg/codegen/testing/test/testdata/unions-inline/schema.json

```diff
@@ -0,0 +1,118 @@
+{
+    "name": "example",
+    "version": "1.0.0",
+    "resources": {
+        "example:index:ExampleServer": {
+            "properties":{
+                "name": {
+                    "type": "string"
+                }
+            },
+            "inputProperties": {
+                "properties": {
+                    "oneOf": [
+                      {
+                        "type": "object",
+                        "$ref": "#/types/example:index:ServerPropertiesForReplica"
+                      },
+                      {
+                        "type": "object",
+                        "$ref": "#/types/example:index:ServerPropertiesForRestore"
+                      }
+                    ],
+                    "discriminator": {
+                      "propertyName": "createMode",
+                      "mapping": {
+                        "PointInTimeRestore": "#/types/example:index:ServerPropertiesForRestore",
+                        "Replica": "#/types/example:index:ServerPropertiesForReplica"
+                      }
+                    }
+                }
+            }
+        }
+    },
+    "types":{
+        "example:index:ServerPropertiesForReplica": {
+          "description": "The properties to create a new replica.",
+          "properties": {
```

Comment:
Can we shrink this, most of these props aren't needed for testing that inline unions work.

Reply:
Done ✅